minion-certificate-manager/main/src/main/java/org/opennms/horizon/minioncertmanager/grpc/MinionCertificateManagerImpl.java

```diff
@@ -76,15 +82,17 @@ public class MinionCertificateManagerImpl extends MinionCertificateManagerGrpc.M
 
     private final SerialNumberRepository serialNumberRepository;
 
+    private final InventoryClient inventoryClient;
+
     private CommandExecutor commandExecutor = new CommandExecutor();
 
     @Autowired
     public MinionCertificateManagerImpl(@Value("${manager.mtls.certificate}") File certificate,
-        @Value("${manager.mtls.privateKey}") File privateKey, @Autowired SerialNumberRepository serialNumberRepository) throws IOException, InterruptedException {
-        this(certificate, privateKey, new PKCS12Generator(), serialNumberRepository);
+        @Value("${manager.mtls.privateKey}") File privateKey, @Autowired SerialNumberRepository serialNumberRepository, @Autowired InventoryClient inventoryClient) throws IOException, InterruptedException {
+        this(certificate, privateKey, new PKCS12Generator(), serialNumberRepository, inventoryClient);
     }
 
-    MinionCertificateManagerImpl(File certificate, File key, PKCS12Generator pkcs8Generator, SerialNumberRepository serialNumberRepository) throws IOException, InterruptedException {
+    MinionCertificateManagerImpl(File certificate, File key, PKCS12Generator pkcs8Generator, SerialNumberRepository serialNumberRepository, InventoryClient inventoryClient) throws IOException, InterruptedException {
```

Comment:
Nit: Line is excessively long and would benefit from being broken into multiple lines.

```suggestion
    MinionCertificateManagerImpl(
        File certificate,
        File key,
        PKCS12Generator pkcs8Generator,
        SerialNumberRepository serialNumberRepository,
        InventoryClient inventoryClient
    ) throws IOException, InterruptedException {
```

Reply:
done